.github/workflows/kit-bootstrap.yml

```diff
@@ -0,0 +1,135 @@
+              else
+                echo "Control plane objects are not ready"
+              fi
+              if [[ "$time" -eq 990 ]]; then
+                echo "Timeout: Control plane objects are not ready after 20 minutes"
+                kubectl logs deploy/kit-controller -n kit
+                exit 1
+              fi
+              sleep 5
+            done
+            kubectl wait --for=condition=Ready pods  --all --timeout=5m
+            kubectl get secret ${GUEST_CLUSTER_NAME}-kube-admin-config -ojsonpath='{.data.config}' | base64 -d > /tmp/kubeconfig
+            kubectl --kubeconfig=/tmp/kubeconfig apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/release-1.10/config/master/aws-k8s-cni.yaml
+            kubectl --kubeconfig=/tmp/kubeconfig rollout status daemonset/aws-node -n kube-system
+            cat <<-EOF | kubectl apply -f -
+              apiVersion: kit.k8s.sh/v1alpha1
+              kind: DataPlane
+              metadata:
+                name: ${GUEST_CLUSTER_NAME}-nodes
+              spec:
+                clusterName: ${GUEST_CLUSTER_NAME} # Associated Cluster name
+                nodeCount: 2
+          EOF
+            for (( time=0; time<=1000; time+=5 )); do
+              dataplane_object=$(kubectl get dataplane ${GUEST_CLUSTER_NAME}-nodes  -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}' 2>/dev/null)
+              if [[ "$dataplane_object" == "True" ]]; then
+                echo "Dataplane objects are ready"
+                break;
+              else
+                echo "Dataplane objects are not ready"
+              fi
+              if [[ "$time" -eq 990 ]]; then
+                echo "Timeout: Dataplane objects are not ready after 10 minutes"
+                kubectl logs deploy/kit-controller -n kit 
+                exit 1
+              fi
+              sleep 5
+            done
+            for (( time=0; time<=300; time+=5 )); do
+              ready_node=$(kubectl --kubeconfig=/tmp/kubeconfig get nodes 2>/dev/null | grep -w Ready | wc -l)
+              if [[ "$ready_node" -eq 2 ]]; then
+                echo "Dataplane nodes are ready"
+                break;
+              else
+                echo "Dataplane nodes are not ready"
+              fi
+              if [[ "$time" -eq 290 ]]; then
+                echo "Timeout: Dataplane nodes are not ready after 10 minutes"
+                exit 1
+              fi
+              sleep 5
+            done
+            kubectl --kubeconfig=/tmp/kubeconfig create deployment nginx-deployment --image=nginx
+            kubectl --kubeconfig=/tmp/kubeconfig rollout status deployment/nginx-deployment
+            kubectl --kubeconfig=/tmp/kubeconfig expose deployment nginx-deployment --name=nginx-service --port=80 --target-port=80 --type=ClusterIP
+            kubectl --kubeconfig=/tmp/kubeconfig certificate approve $(kubectl --kubeconfig=/tmp/kubeconfig get csr | grep Pending | awk '{print $1}')
```

Comment:
This is only useful for local debugging. But it's not useful to run in this test.
What are you trying to do with it?